protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -76,7 +76,9 @@ var wktSchemas = map[string]schemaCore{
 	".google.protobuf.BoolValue": {
 		Type: "boolean",
 	},
-	".google.protobuf.Empty": {},
+	".google.protobuf.Empty": {
+		Properties: &openapiSchemaObjectProperties{},
+	},
```

Comment:
What is the consequence of this change?

Reply:
I think I did this to remove the random if statements on lines 1041 and 1108, but forgot to actually remove them. However still happy to discuss what to do around moving the `Properties` field.